team.html

```diff
@@ -163,6 +413,35 @@ <h5 class="title">
         loadProfiles(); // call function to load profiles
     });
 
+    //function to play the preloader
+    let isLoading = true;
+
+    function preloader() {
+        if (isLoading) {
+            document.getElementById("skeletonAdvisoryPanel").style.display = "block";
+            document.getElementById("skeletonExternalCollaborators").style.display = "block";
+            document.getElementById("skeletonExecutiveCommittee").style.display = "block";
+            document.getElementById("skeletonDevTeam").style.display = "block";
+            document.getElementById("skeletonAlumni").style.display = "block";
+            document.getElementById("advisoryPanel").style.display = "none";
+            document.getElementById("externalCollaborators").style.display = "none";
+            document.getElementById("executiveCommitee").style.display = "none";
+            document.getElementById("techTeam").style.display = "none";
+            document.getElementById("alumni").style.display = "none";
+        } else {
+            document.getElementById("skeletonAdvisoryPanel").style.display = "none";
+            document.getElementById("skeletonExternalCollaborators").style.display = "none";
+            document.getElementById("skeletonExecutiveCommittee").style.display = "none";
+            document.getElementById("skeletonDevTeam").style.display = "none";
+            document.getElementById("skeletonAlumni").style.display = "none";
+            document.getElementById("advisoryPanel").style.display = "block";
+            document.getElementById("externalCollaborators").style.display = "block";
+            document.getElementById("executiveCommitee").style.display = "block";
+            document.getElementById("techTeam").style.display = "block";
+            document.getElementById("alumni").style.display = "block";
+        }
+    }
+
```

Comment:
I don't think that we need this mechanism. Just include them inside the desired div itself. Moustache will replace the content.

ex: 
```
<div id="alumni">
   // skeleton goes here
</div>
```

Any thoughts @Gravewalker666?

---------------
btw,  
`document.getElementById("alumni").style.display = "none";` is equal to `$('#alumni').hide();`

Reply:
Yup, I agree

Reply:
@gajindubandara take a look at this 
https://github.com/Gravewalker666/grave-web/blob/22c44a254d6f47f4cdec03af96d82466a6cb78d1/src/index.js#L67-L86

Reply:
Corresponding html, 
https://github.com/Gravewalker666/grave-web/blob/22c44a254d6f47f4cdec03af96d82466a6cb78d1/src/index.html#L237-L251